### PR TITLE
Upload conditions bug

### PR DESCRIPTION
**Pull request by CodingCarlos**
Fixed issues detected duting qa related to uploads without title, description & product type

### CODE REVIEW

File: languages/es_es.js [hunk 1]
```diff
@@ -58,6 +58,7 @@
 			UPLOAD_TITLE_PLACEHOLDER: 'Título',
 			UPLOAD_DESCRIPTION_PLACEHOLDER: 'Descripcion',
 			UPLOAD_SEND: 'Subir video',
```
Comment (javierLiarte) [inline languages/es_es.js]: check both "video" str

File: pages/upload/upload.controller.js [hunk 2]
```diff
@@ -47,28 +48,39 @@ function UploadController(api, session, $state, $translate) {
...
 				self.loading = false;
 				$state.go('gallery');
 			});
 		} else {
-			self.error = 'No video selected or data is not set';
+			self.error = $translate.instant('UPLOAD_ERROR_DATA');
 		}
 	}
+
+	function canUpload() {
```
Comment (javierLiarte) [inline pages/upload/upload.controller.js:77]: great for the fix! :+1: 

not sure if product types were to be mandatory in the end, please check it with @PabloFMM

**Review by javierLiarte — APPROVED** [review]
Seems all good :)

Just check about the definitive mandatory video params as I'm not sure about all of them now

**Review by DevStarlight — APPROVED** [review]
I didn't know about this bug, but reading the code looks like it was something related to the possibility of upload without required fields like title, description and product type. I've updated the comment of the PR so everybody can understand more about the issue. Regardless of that, good job.

**Pull request merged**